app/src/main/res/menu/menu.xml

```diff
@@ -17,6 +17,11 @@
         android:icon="@drawable/twotone_delete_white_18dp"
         android:title="@string/Clear"
         app:showAsAction="ifRoom" />
+    <item
+        android:id="@+id/loadId"
+        android:icon="@drawable/twotone_save_white_18dp"
```

Comment:
Perhaps we should use different icon from one we use for saving drawings to the gallery

Reply:
updated